lib/modelManagers/articlemodel.js

```diff
@@ -108,7 +116,96 @@ class articleModelManager {
     const returnedArticles = await Articles.findAll(options);
     return returnedArticles;
   }
-}
 
+  /**
+ *
+ *
+ * @static
+ * @param {where} where
+ * @param {value} value
+ * @param {limit} limit
+ * @param {offset} offset
+ * @memberof articleModelManager
+ * @returns {object} Articles by title
+ */
+  static async getArticlesByTitle(where, value, limit, offset) {
+    const options = {
+      include: [includeUser],
```

Comment:
It is because this is just the result of a search, when the article is clicked, then all the tags will be included. It is just to minimize the amount of information being sent back to the user